examples/tcp-retransmit.bpf.c

```diff
@@ -0,0 +1,84 @@
+#include <vmlinux.h>
+#include <bpf/bpf_tracing.h>
+#include <bpf/bpf_core_read.h>
+#include "string.h"
+#include "maps.bpf.h"
+
+#define MAX_ENTRIES 8192
+#define RETRANSMIT  1
+#define TLP         2
+#define AF_INET		2
+#define AF_INET6	10
+
+struct ipv4_key_t {
+	u32 saddr;
+	u32 daddr;
+	__u16 sport;
+	__u16 dport;
+	u32 type;
+};
+
+struct ipv6_key_t {
+	u8 saddr[16];
+	u8 daddr[16];
+	__u16 sport;
+	__u16 dport;
+	u32 type;
+};
+
+struct {
+	__uint(type, BPF_MAP_TYPE_LRU_HASH);
+	__uint(max_entries, MAX_ENTRIES);
+	__type(key, struct ipv4_key_t);
+	__type(value, u64);
+} tcp_retransmit_ipv4_packets_total SEC(".maps");
+
+struct {
+	__uint(type, BPF_MAP_TYPE_LRU_HASH);
+	__uint(max_entries, MAX_ENTRIES);
+	__type(key, struct ipv6_key_t);
+	__type(value, u64);
+} tcp_retransmit_ipv6_packets_total SEC(".maps");
+
+static int trace_event(const struct sock *sk, u32 type)
+{
+	__u32 family = sk->__sk_common.skc_family;
```

Comment:
Why `__u32` rather than `u32`?

Reply:
Should I use type definition always without `__` ?